common/src/main/java/com/microsoft/identity/common/internal/migration/TokenCacheItemMigrationAdapter.java

```diff
@@ -127,25 +128,26 @@
      *
      * @param clientId    String of the given client id.
      * @param redirectUri redirect url string of the given client id.
-     * @param cacheRecord Foci cache record.
+     * @param accountRecord account record of request
+     * @param refreshTokenRecord : refresh token record of FOCI account
      * @return true if the given client id can use the cached foci token. False, otherwise.
      * @throws ClientException
      * @throws IOException
      */
-    public static boolean tryFociTokenWithGivenClientId(@NonNull final BrokerOAuth2TokenCache brokerOAuth2TokenCache,
+    public static boolean tryFociTokenWithGivenClientId(@NonNull final OAuth2TokenCache brokerOAuth2TokenCache,
                                                         @NonNull final String clientId,
                                                         @NonNull final String redirectUri,
-                                                        @NonNull final ICacheRecord cacheRecord)
+                                                        @NonNull final RefreshTokenRecord refreshTokenRecord,
```

Comment:
API breaking change FYI

Would it be better to make an additive change here?

Reply:
yup, `broker` uses it too. I made changes on broker , shouldn't have an impact,  but I think it is better to add as an additive to be safe

Reply:
added as an additive change